### ISSUE TITLE
Can't load anything (Genres, Featured Playlists, Search etc.)

### ISSUE THREAD

**Issue by hughwilliams94** [jaedb/Iris#404]
**Describe the bug**
Nothing will load in Iris

**To Reproduce**
1. Start Mopidy with Mopidy-Spotify and Mopidy-Iris
2. Click on basically anything
3. See error: `Could not load featured playlists` etc.

**Expected behavior**
The selected menu should load with tracks. albums, artists or whatever

**Environment details:**
 - OS: Arch Linux
 - Browser Firefox
 - Iris version 3.37.1 (Installed from AUR)

**Additional context**
Mopidy and Mopidy-Spotify are working fine as I can search freely using mopidy and ncmpcpp. Everything in Iris seems fine except that under the Spotify tab (which reads Authorized) the Current User is listed as unknown (this may be leading to the `Could not load your profile` errors

**Comment by jaedb** [comment]
Are you still experiencing the issue? I do notice that Spotify's HTTP API does go down from time-to-time, requiring a manual re-Authorization. Can you try this (ie _Settings > Spotify > Log out_ then _Log in_) to see if it helps?

**Comment by KillerX** [comment]
I am experiencing this exact thing. One interesting note:   
`https://api.spotify.com/v1/search?q=%22test%22&type=album%2Cartist%2Ctrack&limit=50`   
results in `502 - The server encountered a temporary error and could not complete your request.` 

While   
`https://api.spotify.com/v1/search?q=test&type=album%2Cartist%2Ctrack&limit=50`   
correctly results in

```json
{
  "error": {
    "status": 401,
    "message": "No token provided"
  }
}
```

The difference is the quote marks around the value of `q=` parameter.

**Comment by jaedb** [comment]
That is interesting. I wonder where those quotemarks are being injected.

In any case, the fact you're getting a _502_ error from Spotify tells us the issue is at Spotify's end. Nearly a week has passed since you observed this, does it still persist?

**Comment by KillerX** [comment]
At this point it appears that the search has returned to a somewhat working condition, but my mopidy is segfaulting now :sweat_smile: (nothing to do with iris presumably). I suggest to consider this resolved and we can open a new issue if it comes up again.

**Comment by PhilippJust** [comment]
Just in case anyone else encounters this while googling:
I experienced the same behaviour. It was caused by a privacy browser plugin blocking `api.spotify.com`. Obviously that's not what we want :) So if you experience this, checking if network calls are blocked by plugins is worthwhile.

**Comment by Squidpower** [comment]
> Just in case anyone else encounters this while googling:
> I experienced the same behaviour. It was caused by a privacy browser plugin blocking `api.spotify.com`. Obviously that's not what we want :) So if you experience this, checking if network calls are blocked by plugins is worthwhile.

Thank you so much! I was going crazy over this one!

**Comment by jaedb** [comment]
@Squidpower and @PhilippJust what browser extension(s) were to blame? It would be quite useful for other users experiencing this issue.

**Comment by PhilippJust** [comment]
> @Squidpower and @PhilippJust what browser extension(s) were to blame? It would be quite useful for other users experiencing this issue.

You're right. Updated my comment above.
